src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -1986,6 +2001,42 @@ otError DBusThreadObjectRcp::SetEphemeralKeyEnabled(DBusMessageIter &aIter)
     return error;
 }
 
+void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aRequest)
+{
+    otError error        = OT_ERROR_NONE;
+    auto    threadHelper = mHost.GetThreadHelper();
+
+    otBorderAgentClearEphemeralKey(threadHelper->GetInstance());
+    aRequest.ReplyOtResult(error);
+}
+
+void DBusThreadObjectRcp::ActivateEphemeralKeyModeHandler(DBusRequest &aRequest)
+{
+    otError     error        = OT_ERROR_NONE;
+    auto        threadHelper = mHost.GetThreadHelper();
+    uint32_t    lifetime     = 0;
+    uint16_t    port         = OTBR_CONFIG_BORDER_AGENT_MESHCOP_E_UDP_PORT;
+    auto        args         = std::tie(lifetime);
+    std::string ePSKc;
```

Comment:
nit: I think the name style should make it `ePskc` but I may be wrong though.

Reply:
+1. Yes. `ePskc` would be recommended and it follows the established naming style.

We use `CamelCase` and treat abbreviations as a single word within that style. Examples: `otUdpSocket`, `otSrpClient`, `otMleCounters`, etc.

Reply:
Thanks! Accept.